TaylorIntegrationTests/ReporterComparator.swift

```diff
@@ -33,14 +33,14 @@ func !=(lhs: [String:AnyObject], rhs: [String:AnyObject]) -> Bool {
 
 class ReporterComparator {
     
-    func compareReporters(_ firstReporterPath: String, secondReporterPath: String) -> Bool {
-        var violations1 = JSONToViolationParser().parseFile(firstReporterPath)
+    func compareReporter(atPath path1: String, withReporterAtPath path2: String) -> Bool {
```

Comment:
Then, maybe will be better to name it `compareReport(atPath:withReportAtPath:)` aka **report** and not **reporter**?